testthing/test.go

```diff
@@ -0,0 +1,349 @@
+////////////////////////////////////////////////////////////////////////////////
```

Comment:
fine keeping this file, just change the folder name to `debugging_tools`

Reply:
🆗